CommandLine/emake/Proto2ES.cpp

```diff
@@ -237,13 +237,15 @@ std::string Actions2Code(const ::google::protobuf::RepeatedPtrField< buffers::re
 
 #include <zlib.h>
 
-unsigned char* zlib_compress(unsigned char* inbuffer,int actualsize)
+unsigned char* zlib_compress(unsigned char* inbuffer,int &actualsize)
 {
     uLongf outsize=(int)(actualsize*1.1)+12;
```

Comment:
So this is what you were referring to. This was because, if you pass zlib mostly-compressed (or mostly-noisy) data, the final result can be upwards of 10% bigger than the original data. The header is 12 bytes.